lib/transforms/minifyCss.js

```diff
@@ -15,10 +15,6 @@ module.exports = queryObj => {
                     'default',
                     {
                         svgo: false,
-                        safe: true,
-                        normalizeUrl: false, // Buggy
-                        mergeLonghand: false,
-                        autoprefixer: false,
```

Comment:
Does this removal mean autoprefixer will be run?

Reply:
No, autoprefixer is not included in the `default` preset: https://github.com/ben-eb/cssnano/blob/af7c3f4f8a31b9356cf69daf88cd7da56357b633/packages/cssnano-preset-default/src/index.js